### PR TITLE
Add service icons

### PR DESCRIPTION
**Pull request by epwinchell**
Screen Shot 2023-02-01 at 4 12 40 PM

### CODE REVIEW

**Comment by epwinchell** [comment]
@josejulio Is there a way we can use `GenericIcon` as a fall back?

**Comment by josejulio** [comment]
> @josejulio Is there a way we can use `GenericIcon` as a fall back?

We could set it as optional and use the GenericIcon if is undefined.

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -1,23 +1,35 @@
 import {FunctionComponent} from 'react';
-import {Card as PFCard, CardBody, Text, TextContent, TextVariants} from '@patternfly/react-core';
+import {Card as PFCard, CardBody, Split, SplitItem, Text, TextContent, TextVariants} from '@patternfly/react-core';
+
+import APIConfigurationIcons from '../../config/APIConfigurationIcons';
 
 export interface CardProps {
   displayName: string;
+  icon: keyof typeof APIConfigurationIcons;
```
Comment (josejulio) [inline src/components/Card/Card.tsx:8]:
```suggestion
  icon?: keyof typeof APIConfigurationIcons;
```

File: src/pages/LandingPage.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ export const LandingPage: FunctionComponent = () => {
         <Gallery hasGutter>
           { apiConfigurations.map(apiConfig => (
             <GalleryItem key={apiConfig.displayName}>
-              <Card displayName={apiConfig.displayName} description={apiConfig.description} onClick={() => navigate(`/api/${apiConfig.displayName}`)} />
+              <Card displayName={apiConfig.displayName} icon={apiConfig.icon} description={apiConfig.description} onClick={() => navigate(`/api/${apiConfig.displayName}`)} />
```
Comment (josejulio) [inline src/pages/LandingPage.tsx:73]:
```suggestion
              <Card displayName={apiConfig.displayName} icon={apiConfig.icon ?? APIConfigurationIcons.GenericIcon} description={apiConfig.description} onClick={() => navigate(`/api/${apiConfig.displayName}`)} />
```

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**